tests/unit/test_dto/test_factory/test_integration.py

```diff
@@ -29,7 +27,7 @@
 
 
 def test_url_encoded_form_data(use_experimental_dto_backend: bool) -> None:
-    @dataclass
+    @dataclass()
```

Comment:
Just a nit, but why braces for these decorators and not others?

Reply:
To be honest, I have no idea. Mypy was complaining when the braces were missing here :shrug:

Reply:
🤷‍♂️